### PR TITLE
fix syntax and semantic issues in vmi-webhook and config-test

### PR DESCRIPTION
**Pull request by alonSadan**
**What this PR does / why we need it**:
This PR fixes some typo/syntax/semntical issues in the vmi
addmition webhook and configuration test raised here:
https://github.com/kubevirt/kubevirt/pull/2493

**Special notes for your reviewer**:
The changes were suggested in the attached PR 
**Release note**:

```release-note

```

### CODE REVIEW

File: pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go [hunk 1]
```diff
@@ -554,27 +554,27 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 			if cniTypesCount == 0 {
 				causes = append(causes, metav1.StatusCause{
 					Type:    metav1.CauseTypeFieldValueRequired,
-					Message: fmt.Sprintf("should have a network type"),
```
Comment (fromanirh) [inline pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go]: hey, could you please elaborate why `fmt.Sprintf` is needed here?

Comment (alonSadan) [reply pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go]: missed that. Thanks

File: pkg/virt-config/config_test.go [hunk 2]
```diff
@@ -133,8 +133,8 @@ var _ = Describe("ConfigMap", func() {
 		cpuRequest := clusterConfig.GetCPURequest()
 		Expect(cpuRequest.String()).To(Equal(result))
 	},
-		table.Entry("when set, it should return the value", "400m", "400m"),
-		table.Entry("when unset, it should return the default", "", virtconfig.DefaultCPURequest),
+		table.Entry("when set, GetCPURequest().string() should return the value", "400m", "400m"),
```
Comment (SchSeba) [inline pkg/virt-config/config_test.go]: I think this should be `GetCPURequest`

**Comment by slintes** [comment]
thanks!

/approve

**Pull request merged**